index.html

```diff
@@ -0,0 +1,170 @@
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Quiz</title>
+	<link rel="stylesheet" type="text/css" href="reset.css">
+	<link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+	<div id="container">
+		<article id="product1" class="productCard">
+			<header class="productHeader">
+				<h1>Adorable Kitten</h1>
+			</header>
+			<section class="productInfo1">
+				<img src="kitten.jpeg" alt="This is an image">
+				<p class="productDesc">If it fits, i sits cat is love, cat is life. Play riveting piece on synthesizer keyboard leave hair everywhere intently sniff hand kick up litter. Kitty loves pigs ignore the squirrels, you'll never catch them anyway yet stand in front of the computer screen. Lick the plastic bag you call this cat food? knock over christmas tree. Swat at dog Gate keepers of hell and chew on cable. Eat from dog's food meow all night having their mate disturbing sleeping humans, and the dog smells bad yet chase laser for leave dead animals as gifts, yet asdflkjaertvlkjasntvkjn (sits on keyboard) meowing non stop for food.</p>
+				<p class="productAvail">Available</p>
+			</section>
+			<section class="productInfo2">
+				<h2>Specifications</h2>
+				<p class="productSize">Size: 8"</p>
+				<p class="productWeight">Weight: 1.4lbs</p>
+				<p class="productValidUntil">Specifications valid until next week</p>
+			</section>
+			<section class="productInfo3">
+				<header class="pricingHeader">
+					<h2>Pricing</h2>
+				</header>
+				<p class="pricing1">1: $15</p>
+				<p class="pricing2">2-10: $10</p>
+				<p class="pricing3">11+: Why are you buying kittens in bulk?</p>
+			</section>
+		</article>
+
+
+		<article id="product2" class="productCard">
+			<header class="productHeader">
+				<h1>Funny Kitten</h1>
+			</header>
+			<section class="productInfo1">
+				<img src="kitten.jpeg" alt="This is an image">
+				<p class="productDesc">If it fits, i sits cat is love, cat is life. Play riveting piece on synthesizer keyboard leave hair everywhere intently sniff hand kick up litter. Kitty loves pigs ignore the squirrels, you'll never catch them anyway yet stand in front of the computer screen. Lick the plastic bag you call this cat food? knock over christmas tree. Swat at dog Gate keepers of hell and chew on cable. Eat from dog's food meow all night having their mate disturbing sleeping humans, and the dog smells bad yet chase laser for leave dead animals as gifts, yet asdflkjaertvlkjasntvkjn (sits on keyboard) meowing non stop for food.</p>
+				<p class="productAvail">Available</p>
+			</section>
+			<section class="productInfo2">
+				<h2>Specifications</h2>
+				<p class="productSize">Size: 8"</p>
+				<p class="productWeight">Weight: 1.4lbs</p>
+				<p class="productValidUntil">Specifications valid until next week</p>
+			</section>
+			<section class="productInfo3">
+				<header class="pricingHeader">
+					<h2>Pricing</h2>
+				</header>
+				<p class="pricing1">1: $15</p>
```

Comment:
Same advice as previously about naming classes with their existing sequence in the HTML